### PR TITLE
fix: tx.IterateBuckets cannot iterate.

### PR DESCRIPTION
**Pull request by diiyw**

### CODE REVIEW

**Comment by bigboss2063** [comment]
Hi, please open a issue and describe this bug, thanks. @diiyw

**Comment by diiyw** [comment]
> Hi, please open a issue and describe this bug, thanks. @diiyw

Yeah, here :  https://github.com/nutsdb/nutsdb/issues/544

File: tx_bucket.go [hunk 1]
```diff
@@ -20,20 +20,16 @@ func (tx *Tx) IterateBuckets(ds uint16, pattern string, f func(bucket string) bo
 		return err
 	}
 
-	handle := func(bucket string) error {
-		if end, err := MatchForRange(pattern, bucket, f); end || err != nil {
-			return err
-		}
-		return nil
-	}
 	var err error
```
Comment (bigboss2063) [inline tx_bucket.go:23]: I thinks this err could be removed and just return nil in the end.

Comment (diiyw) [reply tx_bucket.go]: Okay, I've fixed it.

**Review by bigboss2063 — APPROVED** [review]
LGTM. Thanks for your great work.

**Pull request merged**